scripts/base/init-bare.zeek

```diff
@@ -5496,6 +5496,17 @@ export {
 	const flowbuffer_contract_threshold = 2 * 1024 * 1024 &redef;
 }
 
+module Pcap;
```

Comment:
There's already a module Pcap on line 5121 further up. Any reason it couldn't be there?

Reply:
Nope, I just didn't look to see if it was already there.